Formula/n/neosync.rb

```diff
@@ -0,0 +1,31 @@
+class Neosync < Formula
+  desc "CLI for interfacing with Neosync"
+  homepage "https://www.neosync.dev/"
+  url "https://github.com/nucleuscloud/neosync/archive/refs/tags/v0.3.36.tar.gz"
+  sha256 "c3afe66bc90de2ee1f4c6d88df78d50a9f9e6af41b85ec1675fdf1ead8c6b4fa"
+  license "MIT"
+  head "https://github.com/nucleuscloud/neosync.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X github.com/nucleuscloud/neosync/cli/internal/version.gitVersion=#{version}
+      -X github.com/nucleuscloud/neosync/cli/internal/version.gitCommit=#{tap.user}
+      -X github.com/nucleuscloud/neosync/cli/internal/version.buildDate=#{time.iso8601}
+    ]
+    cd "cli" do
+      system "go", "build", *std_go_args(ldflags: ldflags), "./cmd/neosync"
+    end
```

Comment:
```suggestion
    system "go", "build", *std_go_args(ldflags: ldflags), "./cli/cmd/neosync"
```

Reply:
Does this work?

Reply:
it wont work, the go.mod is in the cli folder.